tests/integration/it_scheme_types_test.py

```diff
@@ -31,3 +33,18 @@ def test_scheme_condition_list(client, db_session):
                                     scheme_type_id=scheme_type.id))
     assert conditions.status_code == HTTPStatus.OK
     assert b"Test condition" in conditions.data
+
+
+def test_scheme_condition_create(client, db_session):
+    """ Test that CreateView works correctly and creates an instance """
+    scheme_type = factories.SchemeTypeFactory()
```

Comment:
@vryazanov according to your test, it's create action. But actually here is not create here, it looks like edit. Is it ok?

Reply:
@vladarefiev test is about creation of `SchemeTypeCondition`, not `SchemeType`.  `scheme_type` is just a helper, it's needed in url path